kubeadm/scripts/CleanupNode.ps1

```diff
@@ -0,0 +1,28 @@
+if (Get-Service kubelet -ErrorAction Ignore) {
+    Stop-Service kubelet
+    sc.exe delete kubelet
+}
+if (Get-Service rancher-wins -ErrorAction Ignore) {
+    Stop-Service rancher-wins
+    sc.exe delete rancher-wins
+}
+if (Get-Service docker -ErrorAction Ignore) {
+    Restart-Service docker
+}
+if (Get-Service containerd -ErrorAction Ignore) {
+    Restart-Service containerd
+}
+Stop-Process -Name rancher-wins* -Force
```

Comment:
Isn't the executable just `wins.exe` and the process name therefore `wins`?
```suggestion
Stop-Process -Name wins -Force
```

Reply:
Unfortunately no. In this case we try to cleanup this processes:
```
PS C:\Users\v.leschenko> Get-Process rancher-wins*

Handles  NPM(K)    PM(K)      WS(K)     CPU(s)     Id  SI ProcessName
-------  ------    -----      -----     ------     --  -- -----------
    228      15    21716      22048     484.08  16672   0 rancher-wins-flanneld
    231      19    29024      30968   1,583.41   8896   0 rancher-wins-kube-proxy
```

Sometimes when something happens this processed running even after rancher-wins service was removed.